src/main/java/de/retest/web/WebElementPeer.java

```diff
@@ -100,14 +100,21 @@ public OutlineAttribute retrieveOutline() {
 	protected MutableAttributes retrieveStateAttributes() {
 		final MutableAttributes state = new MutableAttributes();
 		for ( final String attribute : AttributesProvider.getInstance().getAttributes() ) {
-			final String attributeValue = webData.get( attribute );
+			final String attributeValue = normalize( webData.get( attribute ) );
```

Comment:
Shouldn't we do this to all attributes (i.e inside the constructor)?

Reply:
Not so sure about this. I don't know how future Attributes will be implemented and for which technologies. Would refrain from generalizing too much...